src/js/state/Configs/index.ts

```diff
@@ -0,0 +1,42 @@
+import {createEntityAdapter, createSlice} from "@reduxjs/toolkit"
+import {State} from "../types"
+
+export type ConfigItemType = "file" | "string" // | "number" | "boolean"
+
+export type ConfigItem = {
+  name: string
```

Comment:
Maybe this name doesn't need to be stored? Since the key and the name are the same and should be in sync?